extensions/http/src/main/java/org/mule/extension/http/api/HttpPart.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.http.api;
+
+import org.mule.runtime.extension.api.annotation.Alias;
+import org.mule.runtime.extension.api.annotation.Parameter;
+import org.mule.runtime.extension.api.annotation.param.Optional;
+
+/**
+ * Representation of an HTTP attachment.
+ */
+@Alias("part")
+public class HttpPart
```

Comment:
missing `@since` and all parameter fields should have a javadoc so that the XSD gets documented